defaults/main.yml

```diff
@@ -13,6 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Wheter to add {{ ansible_managed }} information on top of keepalive.conf
```

Comment:
I am sorry I missed this in my first review:
Can you:
1) Update the comment to mention keepalived.conf instead of keepalive.conf
2) Make it `false` by default?

The reason to make it false would be that the existing users who would update the role would not have a "changed" behaviour, which would restart keepalived, for something they didn't opt-in.

Sorry for the inconvenience.

Reply:
I fully understand your argument and i absoluty agree with.

Reply:
Thanks for your understanding :)